blocks/lead-art-block/features/leadart/default.test.jsx

```diff
@@ -196,7 +197,12 @@ describe("LeadArt", () => {
 			},
 		};
 		useFusionContext.mockReturnValue({ ...fusionContext, globalContent });
-		const { queryByRole, queryByText } = render(<LeadArt customFields={{}} />);
+		const testRender = render(<LeadArt customFields={{}} />);
+		const { queryByRole, queryByText } = testRender;
+
+		EventEmitter.dispatch("galleryExpandEnter");
+		EventEmitter.dispatch("galleryExpandExit");
+
```

Comment:
Is this `EventEmitter` code still necessary in the tests?

Reply:
No, it not I forgot to remove it